sdks/python/tox.ini

```diff
@@ -33,6 +33,16 @@ extras = test
 whitelist_externals =
   find
   time
+deps =
+  grpcio-tools==1.3.5
+  cython: cython==0.26.1
+
+# These 2 magic command overrides are required for Jenkins builds.
```

Comment:
This is bizarre and a good find. I wonder if it is any better with the new tox 3 (released on 4/2).

Reply:
I believe this is already running using tox 3.0.0.